### ISSUE TITLE
Make site accessible 

### ISSUE THREAD

**Issue by samatar26** [yvonne-liu/week6-soFly-FACt#37]

**Comment by samatar26** [comment]
All links should have link text

**Comment by samatar26** [comment]
All images should have alt text
